pom.xml

```diff
@@ -123,6 +124,12 @@
             <version>1.1</version>
         </dependency>
 
+        <dependency>
+            <groupId>org.jenkins-ci.plugins</groupId>
+            <artifactId>token-macro</artifactId>
```

Comment:
Lot of plugins uses token macro. Don't want to maintain one more option and handle functionality of token-macro